src/microbe_stage/organelle_components/ExternallyPositionedComponent.cs

```diff
@@ -12,6 +13,7 @@ public abstract class ExternallyPositionedComponent : IOrganelleComponent
     /// </summary>
     protected static readonly Vector3 DefaultVisualPos = Vector3.Forward;
 
+    [JsonProperty]
```

Comment:
Why was this added? I feel like this change has pretty huge implications. All the components should be reattached to the microbe anyway when loading a save.

Reply:
That may be a holdover from the temporary save fix I made that I forgot to delete. I'll remove it and see what happens.

Reply:
Yep, I think that was it. No issues with saves after deleting it, so I've done that.